_info/Tribute

```diff
@@ -1,6 +1,7 @@
 Tribute
 
-Every morning, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. The team which pays the least money for their tribute, will have their tribute killed. Use `!tribute` to set your tribute candidate if you're the king. 
+Every morning, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. 
+The team which pays the least money for their tribute, will have their tribute killed. Use `!tribute` to set your tribute candidate if you're the king. 
```

Comment:
unnecessary comma here for (grammar issue). Also, the phrasing is a very long way of saying lowest paying team

Reply:
Agreed